src/reducers/requests/admin.ts

```diff
@@ -408,6 +408,16 @@ function disablePlugin(state: RequestStatusType = initialRequestState(), action:
     );
 }
 
+function sendWarnMetricAck(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
```

Comment:
We are no longer tracking the request unless is strictly needed, for example use the request completion to trigger something in other side of the code. But if the code making the call is the one that is reacting to the changes we only need to use the async/await (or promise) result for showing that.

Reply:
Removing this is making some changes here and there, not only this reducer.

Reply:
removed the request tracking defines+related.